vulnerabilities/importers/apache_httpd.py

```diff
@@ -40,7 +48,12 @@ class ApacheHTTPDDataSourceConfiguration(DataSourceConfiguration):
 class ApacheHTTPDDataSource(DataSource):
 
     CONFIG_CLASS = ApacheHTTPDDataSourceConfiguration
-    url = "https://httpd.apache.org/security/vulnerabilities-httpd.xml"
+    url = "https://httpd.apache.org/security/json/{}"
```

Comment:
Maybe use urllib here  instead ? That would be more concise in the code. 

eg you could have something like:
```python
base_url = "https://httpd.apache.org/security/json/"
```
Then in `fetch_links` you could obtain full urls by doing something like
```
links.append(urllib.parse.urljoin(self.base_url, link))
```